make_statistic_plots.py

```diff
@@ -0,0 +1,582 @@
+                "Louisiana State":"United States",
+                "Harper College":"United States",                           
+                "Chicago":"United States",   
+                "Texas":"United States",
+                "American Museum of Natural History":"United States",  
+                "SETI":"United States",
+                "Ames":"United States",
+                "Space Telescope Science Insitute":"United States",
+                "Annapolis":"United States",
+                "Goddard":"United States",
+                "University of Maryland":"United States",
+                "Fisk University":"United States",
+                "Budapest":"Hungary",
+                "Santiago":"Chile",
+                "Goettingen":"Germany",
+                "Korea":"South Korea", #hot take
+                #Crimea is also on the list. Not sure how to handle this. 
+                "University of Southampton":"United Kingdom",
+                "University of St Andrews":"United Kingdom",
+                "University of Leicester":"United Kingdom",
+                "University College London":"United Kingdom",
+                "University of Oxford":"United Kingdom",
+                "University of Manchester":"United Kingdom",
+                "Geneva":"Switzerland",
+                "Zurich":"Switzerland",
+                "New South Wales":"Australia",
+                "Tokyo":"Japan",
+                "Akdeniz University":"Turkey",
+                "University of Colombo":"Sri Lanka",
+                "Berlin":"Germany"
+                          }
+
+#Many affiliations just contain the University or organization, so this links them to the state
+common_mistakes_state = {"Princeton":"New Jersey",
+                        "Goddard":"Maryland",
+                        "GSFC":"Maryland",
+                        "Noqsi":"Colorado",
+                        "Lowell":"Arizona",
+                        "Ames":"California",
+                        "UCB Space Sciences Lab":"California", #Berkeley
+                        "Cornell":"New York",
+                        "Space Telescope Science Institute":"Maryland",
+                        "Lehigh":"Pennsylvania",
+                        "Vanderbilt":"Tennessee",
+                        "SETI Institute":"California",
+                        "Hawai'i":"Hawaii",
+                        "Mikulski":"Maryland",
+                        "Kitt Peak":"Arizona",
+                        "Adler Planetarium":"Illinois",
+                        "American Museum of Natural History":"New York",
+                        "Harvard-Smithsonian":"Massachusetts",
+                        "San Diego":"California",
+                        "Chicago":"Illinois",
+                        "Fisk University":"Tennessee",
+                        "Space Telescope Science Insitute":"Maryland"
+                        }
+
+#Some functions to get the demographic info
+def findCountry(affil):
+    
```

Comment:
Because we might want to reuse these functions, can you add docustrings? It's a good habit to get into. Let's use the [`numpydoc` style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html). Here your docstring might look like:
```
Function to find country given an affiliation 

Parameters
------------
affil: str
    Affiliation, a sentence as a string.
Returns
--------
country: str or None
    A country extracted from the affiliation compliant with XXXX, if one can be found. Otherwise, will return None.
```